docs/configuration.md

```diff
@@ -168,3 +168,20 @@ git config --global credential.gitHubAuthModes "oauth basic"
 ```
 
 **Also see: [GCM_GITHUB_AUTHMODES](environment.md#GCM_GITHUB_AUTHMODES)**
+
+---
+
+### credential.namespace
```

Comment:
Did this need to be in this PR, or could it be a different one?

Reply:
The `namespace` support is needed to maintain back-compat with older GCM's credentials (they're all prefixed with `git:`). We've pushed the logic for building the names into the credential stores, rather than the host providers (who used to just always prepend `git:`).

We could hardcode `git:`, only to then pull it out in a subsequent PR as a setting of course?